LICENSE

```diff
@@ -334,4 +334,24 @@ This product includes code from Apache Flink.
 
 Copyright: 1999-2022 The Apache Software Foundation.
 Home page: https://flink.apache.org/
-License: https://www.apache.org/licenses/LICENSE-2.0
\ No newline at end of file
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes software developed at The Apache Software Foundation.
+
+Apache Kafka (kafka-clients, kafka-connect-api, kafka-connect-json, kafka-connect-transforms)
+
+Copyright: 1999-2022 The Apache Software Foundation.
+Home page: https://kafka.apache.org/
+License: https://www.apache.org/licenses/LICENSE-2.0
+
+--------------------------------------------------------------------------------
+
+This product includes software developed by MongoDB, Inc.
+
+* MongoDB BSON (bson)
+
+Copyright: 2008-present MongoDB, Inc.
+Home page: https://www.mongodb.com/
+License: https://github.com/mongodb/mongo-java-driver/blob/main/LICENSE.txt
```

Comment:
@danielcweeks is this look ok?

Reply:
You shouldn't modify the root LICENSE

Reply:
Given this isn't yet being included in the runtime, we don't necessarily need to add anything to the runtime LICENSE until we add it to that.

Reply:
@bryanck should it be the NOTICE file instead?  `bson` is using `implementation` scope? is it then included with the distribution?